lib/models/listings/furniture_listing.js

```diff
@@ -6,7 +6,18 @@ var secure = require('./shared/secure.js');
 var base_listing = require('./listing.js');
 var photo = require('./shared/photo.js');
 var photoSchema = photo.schema;
+var validation = require('../../validation.js');
 
+var FURNITURE_TITLE_LEN_MIN = 1;
```

Comment:
I made the MIN vars all 1 because I felt it was a bit more human-readable, if you will, in that it is the minimum acceptable length for a title. Works well if we decide to make it something greater than 1, which we should since a single letter isn't very descriptive. We should probably make them all like 3 or something like that.